test/valgrind/valgrindTestSuite.sh

```diff
@@ -100,24 +100,6 @@ function vMsg()
 
 
 
-# -----------------------------------------------------------------------------
-#
-# fileCleanup
-#
-function fileCleanup()
```

Comment:
Unused function (I guess it comes from the time .vtest files were used but they were removed time ago).

NTC (just informative)